packages/gravity-ui-web/src/sass/05-components/03-organisms/_hero.scss

```diff
@@ -36,6 +36,7 @@
     }
 
     h1 {
+      max-width: $grav-page-content-max-width - ($container-horizontal-max-margin * 2);
```

Comment:
Why is this needed?

Shouldn't headings just have their default `display: block;` behaviour and fill out whatever horizontal space they're given by their parent?

If a certain consumer of Gravity, like the website, need to tweak things they should do so in their code. Otherwise, we might risk breaking things in other consumers - e.g. Cycle 7 website.

Reply:
Correct, removed.